config/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v2
+name: karpenter
+description: A Helm chart for https://github.com/awslabs/karpenter/.
+type: application
+version: 0.1
```

Comment:
Can you lock this version to the current repo's version v0.2.3?

Reply:
Agreed